dsp/ImpulseResponse.h

```diff
@@ -20,8 +20,11 @@ namespace dsp
 class ImpulseResponse : public History
 {
 public:
+  struct IRData;
```

Comment:
Can we get this removed? Doesn't seem to be doing anything except hanging out on the public interface?